composer.json

```diff
@@ -31,19 +32,21 @@
       }
     },
     "suggest": {
-        "zendframework/zend-expressive-platesrenderer": "^0.1 to use the Plates template renderer",
-        "zendframework/zend-expressive-twigrenderer": "^0.1 to use the Twig template renderer",
-        "zendframework/zend-expressive-zendviewrenderer": "^0.1 to use the zend-view PhpRenderer template renderer"
+        "zendframework/zend-expressive-platesrenderer": "^1.2 to use the Plates template renderer",
+        "zendframework/zend-expressive-twigrenderer": "^1.2.1 to use the Twig template renderer",
+        "zendframework/zend-expressive-zendviewrenderer": "^1.2.1 to use the zend-view PhpRenderer template renderer"
```

Comment:
I'll update each of these on merge, as I've just finished releasing new versions of each.